src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly

```diff
@@ -146,7 +146,11 @@ THE SOFTWARE.
         <j:forEach var="g" items="${groups}">
           <j:forEach var="p" items="${g.permissions}">
             <j:if test="${descriptor.showPermission(p)}">
-              <th class="pane" tooltip="${descriptor.getDescription(p)}">
+              <th class="pane" html-tooltip="${descriptor.getDescription(p)}" tooltip="${descriptor.getDescription(p)}">
+                <!--
+                  Tooltip needs to be HTML for GlobalMatrixAuthorizationStrategy.PermissionNotImpliedBy formatting.
+                  So set 'html-tooltip' for Tippy and 'tooltip' for YUI.
+                 -->
```

Comment:
Can I rely on `html-tooltip` to have precedence over `tooltip` for this situation?

Reply:
no they'll both get set from what I can tell, probably want to correct it in JavaScript.

Reply:
Any suggestions how I can make this compatible both with current Jenkins and future Jenkins?

Reply:
> correct it in JavaScript

^.^

https://github.com/jenkinsci/matrix-auth-plugin/pull/120/commits/3a215df7cecc7a018539fd20b6fdc42fe1b69c47